develop/jail_scripts/c_jail.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+if [ ! -d "c_jail" ]; then
+	mkdir c_jail 
+fi
+
+mkdir ./c_jail/usr
+mkdir ./c_jail/usr/bin
+mkdir ./c_jail/usr/lib
+mkdir ./c_jail/bin
+mkdir ./c_jail/lib64
+mkdir ./c_jail/lib
+mkdir ./c_jail/lib/x86_64-linux-gnu
```

Comment:
Can this be refactored to be like the c++ jail script where the list of directories is an array and there is a loop that iterates through each of those directories?  That will make these scripts more similar and easier to keep up-to-date